components/scripts/local-dev/manifests/frontend-deployment.yaml

```
@@ -0,0 +1,91 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: vteam-frontend
+  labels:
+    app: vteam-frontend
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: vteam-frontend
+  template:
+    metadata:
+      labels:
+        app: vteam-frontend
+    spec:
+      containers:
+      - name: frontend
+        image: image-registry.openshift-image-registry.svc:5000/vteam-dev/vteam-frontend:latest
+        ports:
+        - containerPort: 3000
+          name: http
+        env:
+        - name: NEXT_PUBLIC_BACKEND_URL
+          value: "https://vteam-backend-vteam-dev.apps-crc.testing/api"
+        - name: BACKEND_URL
+          value: "http://vteam-backend:8080/api"
+        - name: NODE_ENV
+          value: "production"
```

```suggestion
          value: "development"
```
this might be a test to see if this gives us more NextJS hot reloading for free?